application/src/main/java/org/togetherjava/tjbot/commands/moderation/MuteCommand.java

```diff
@@ -70,16 +69,17 @@ private static void handleAlreadyMutedTarget(IReplyCallback event) {
         event.reply("The user is already muted.").setEphemeral(true).queue();
     }
 
-    private static RestAction<Boolean> sendDm(ISnowflake target,
-            @Nullable ModerationUtils.TemporaryData temporaryData, String reason, Guild guild,
-            GenericEvent event) {
-        return event.getJDA()
-            .openPrivateChannelById(target.getId())
-            .flatMap(channel -> ModerationUtils.sendDmAdvice(ModerationAction.MUTE, temporaryData,
-                    "This means you can no longer send any messages in the server until you have been unmuted again.",
-                    guild, reason, channel))
-            .mapToResult()
-            .map(Result::isSuccess);
+    private static RestAction<Boolean> sendDm(User target,
+            @Nullable ModerationUtils.TemporaryData temporaryData, String reason, Guild guild) {
+        String durationMessage = temporaryData == null ? "Permanent" : temporaryData.duration();
+        String description =
+                """
+                        Hey there, sorry to tell you but unfortunately you have been muted.
+                        This means you can no longer send any messages in the server until you have been unmuted again.
+                        """;
```

Comment:
please test that the newline behavior is correct here, since u ended `"""` on the next line and not on the same